doc/auth.md

```diff
@@ -76,3 +76,83 @@ where username and password are credentials you intend to use.
 Note that Pravega operator uses `/etc/auth-passwd-volume` as the mounting directory for secrets.
 
 For more security configurations, please check [here](https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md).
+
+Pravega Operator Supports Passing of Auth Parametes as Secret which are mounted as file in both Segment Store and Controller (Operator mounts these secrets in Segementstore pod it's mounted at `/etc/ss-auth-volume` and in Controller pod at `/etc/controller-auth-volume` respectively)
+
+Note that Pravega has to use this feature and start Picking below specified values from file insted of jvm properties which it currently does.
+(This is not implemented at pravega end currently)
+
+Below is how we can create secret and expose them as file for Auth related properties:-
+
+1. Create a File containg `controller.security.auth.delegationToken.signingKey.basis`  as `delegationToken.signingKey.basis` which represent the tokensigning key used to connect to the controller:
+
+Sample encrypted password file:
```

Comment:
Line 89 was maybe a left over from a copy paste from somewhere?

Reply:
I have removed it now.